firefox-ios/Client/Frontend/Browser/BrowserViewController/State/BrowserViewControllerState.swift

```diff
@@ -25,41 +27,44 @@ struct BrowserViewControllerState: ScreenState, Equatable {
         }
 
         self.init(searchScreenState: bvcState.searchScreenState,
-                  usePrivateHomepage: bvcState.usePrivateHomepage,
                   showDataClearanceFlow: bvcState.showDataClearanceFlow,
                   fakespotState: bvcState.fakespotState,
                   toast: bvcState.toast,
                   showOverlay: bvcState.showOverlay,
-                  windowUUID: bvcState.windowUUID)
+                  windowUUID: bvcState.windowUUID,
+                  reloadWebView: bvcState.reloadWebView,
+                  browserViewType: bvcState.browserViewType)
     }
 
     init(windowUUID: WindowUUID) {
         self.init(
             searchScreenState: SearchScreenState(),
-            usePrivateHomepage: false,
             showDataClearanceFlow: false,
             fakespotState: FakespotState(windowUUID: windowUUID),
             toast: nil,
             showOverlay: false,
-            windowUUID: windowUUID)
+            windowUUID: windowUUID,
```

Comment:
curious - why not `reloadWebView` here?

Reply:
It's initialized with a default of false. We only want it to happen when there is a change to the url.